tests/admin_inlines/models.py

```diff
@@ -256,6 +256,23 @@ class SomeChildModel(models.Model):
     parent = models.ForeignKey(SomeParentModel, models.CASCADE)
     readonly_field = models.CharField(max_length=1)
 
+
+# Models for #29961
+class AnotherParent(models.Model):
```

Comment:
Please try to reuse existing models as adding models increases the run time of the test suite. There's always the option to register models with a different site (see `tests/admin_views`), use proxy models, etc.

Reply:
They have now become unneeded with moving the template tests to admin_widgets.